web/app/js/components/Tap.jsx

```diff
@@ -11,6 +11,27 @@ import { httpMethods, processTapEvent, setMaxRps } from './util/TapUtils.jsx';
 import './../../css/tap.css';
 
 const maxNumFilterOptions = 12;
+
+// from https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent
```

Comment:
There's similar websocket error handling in `TopModule.jsx`. It might be good to put this into `TapUtils.jsx` and then we can do the same error handling there.

Reply:
![screen shot 2018-09-12 at 10 09 57 am](https://user-images.githubusercontent.com/236915/45441745-c4dbc980-b674-11e8-8468-8691f1632384.png)